fe/fe-core/src/main/java/com/starrocks/task/AgentBatchTask.java

```diff
@@ -164,7 +164,7 @@ public void run() {
                 for (AgentTask task : tasks) {
                     agentTaskRequests.add(toAgentTaskRequest(task));
                 }
-                client.submit_tasks(agentTaskRequests);
+                client.send_submit_tasks(agentTaskRequests);
```

Comment:
The connection will be reused. If you just send but do not receive a response, when the connection is reused by other client, it may receive a non-corresponding response, which may result in an error.

So I don't think we can change it like this.

If you want to send the request asynchronously, you can use the BRPC protocol to send the request asynchronously and then wait for the response in batch.